packages/components/src/components/hds/badge/types.ts

```diff
@@ -0,0 +1,24 @@
+export type Size = `${SizeValues}`;
```

Comment:
in other PRs (eg. [here](https://github.com/hashicorp/design-system/pull/1984/files#diff-6d495fe2de524a98c786418a10c8399941e00e4048187a6b595529d1fd1eb7e2)) the order of the declarations is first the enum and then the type. Plus the keys here all all UPPERCASE, in the other are PascalCase.

@alex-ju same as above, do you think it would be possible to have a standard "order" (and naming convention) to follow? /cc @hashicorp/hds-engineering